src/requestHandlers/hook.ts

```diff
@@ -168,62 +173,73 @@ const getPanelHooks = async (req: Request, res: Response): Promise<Response> =>
 };
 
 /**
- * Updates a hook with a next panel set
- * @param {number} hook_id ID of hook that is being updated
- * @param {number} panel_set_id ID of panel set that hook links to
- * @returns response or error
+ * Get Panel Hooks request
+ * @param {Request} req 
+ * @param {Response} res 
+ * @returns 
  */
-const _addSetToHookController = (sequelize: Sequelize) => async (hook_id: number, panel_set_id: number) => {
-    try {
-        const panelSet = await getPanelSetByID(sequelize)(panel_set_id);
-        if (panelSet == null) throw new Error('no panel_set exists for given panel_set_id');
-        return await hookService.addSetToHook(sequelize)(hook_id, panel_set_id);
-    }
-    catch (err) {
-        return err;
-    }
-};
-
 const addSetToHook = async (req: Request, res: Response): Promise<Response> => {
     const hook_id = req.body.hook_id;
     const panel_set_id = req.body.panel_set_id;
+    const validateTrunks : boolean = req.body.validateTrunks ?? true;
```

Comment:
I may be misunderstanding, but I don't see the purpose of having the client be able to choose if they override the checks. I see this as causing invalid data and possible infinite loops. Since this request should only be used to hooks panel sets that are already created, shouldn't `validateTrunks` always be true, and no parameter is needed?

Reply:
I did it how we discussed in our meeting, if its always true, it will always fail for publish. As addsettohook is what is used after the publish to hook things up.